chef_master/source/server_security.rst

```diff
@@ -310,6 +310,119 @@ These newer add-ons will also write all of their secrets to ``/etc/opscode/priva
 
 While the file does not contain passwords in plaintext, it is not safe to share with untrusted users. The format of the secrets file allows Chef Infra Server deployments to conform to regulations that forbid the appearance of sensitive data in plain text in configuration files; however, it does not make the file meaningfully more secure.
 
+SSL Encryption Between Chef Infra Server and External PostgreSQL
+================================================================
+
+**New in Chef Infra Server 13.1.13:**  Chef Infra Server 13.1.13 introduces the ability to encrypt traffic between Chef Infra Server and an external PostgreSQL server over SSL.  These instructions are not all-encompassing and assume some familiarity with PostgreSQL administration, configuration, and troubleshooting. Consult the `PostgreSQL documentation <https://www.postgresql.org/docs/9.6/ssl-tcp.html>`_ for more information.
+
+Here is a walkthrough of a typical scenario for enabling encryption between a Chef Infra Server machine and an external PostgreSQL machine.  It is assumed that both machines are networked together and user-accessible.
+
+#. Run the following command on both machines to gain root access:
+
+   .. code-block:: bash
+    
+      sudo -i
+
+#. Ensure that `OpenSSL <https://www.openssl.org>`_ is installed on the PostgreSQL machine.
+
+#. Ensure that PostgreSQL is installed on the PostgreSQL machine and has SSL support compiled-in.  This applies whether you are compiling your own source or using a pre-compiled binary.
+
+#. Place SSL certificates in the proper directories on the PostgreSQL machine, and ensure they have correct filenames, ownerships, and permissions.
+
+#. Enable SSL on PostgreSQL, and specify paths to the SSL certificates.  This can be done by editing ``postgresql.conf`` on the PostgreSQL machine and ensuring the relevant entries are present (substitute the appropriate paths):
```

Comment:
```suggestion
#. Enable SSL on PostgreSQL by editing the ``postgresql.conf`` file. Specify the paths to the SSL certificates and ensure that those files are present on the PostgreSQL machine:
```

Reply:
For context (which may be missing), I wanted to communicate that there's two steps involved:  1) Enable SSL on PostgreSQL (the `ssl=on` part of the file to be edited), and 2) specify paths to the SSL certificates (the remaining part of the file to be edited).  So long as you know that, I'm fine with however you think is best to communicate that (assuming you think it needs to be communicated).  So we can either go with what you have above, add numbers to the steps, like "1) Enable SSL..., and 2) Specify the paths..." or do whatever you think is best.  Let me know and I'll be happy to do it :)

Reply:
We could change it to 

`Enable SSL on PostgreSQL by setting ``ssl = on`` in the ``postgresql.conf`` file. Specify the paths to the SSL certificates in the ``postgresql.conf`` file and ensure that those files are present on the PostgreSQL machine:` 

But I think the two sentences together are a clear indication that there's two things that the user must do with this file.

Reply:
I believe the last sentence ('Also ensure that those certificates are present...') is covered in the previous sentence directly above ('Place SSL certificates in the proper directories...'), so you could probably eliminate that sentence or fold that edit into the previous sentence above it.

Reply:
Sounds good to me.